sumit-jr/Airbnb
# Update paths for login, sign-up, logout, and logo.

### Merge Request Description

#### **Context**  
This MR updates the paths for login, sign-up, logout, and the logo to ensure consistency and proper functionality across the application. The changes were necessary to align with recent updates to the routing system and to fix issues with the favicon not loading correctly.

---

#### **Changes Included**  
1. **Updated Paths for Authentication**  
   - Modified _header.html.erb


---

#### **Testing**  
- **Manual Testing**  
  - Verified that login, sign-up, and logout links work as expected.  
  - Confirmed that when clicked on logo it successfully go to root
